test/crio-wipe.bats

```diff
@@ -32,35 +32,24 @@ function run_crio_wipe() {
 # test_crio_wiped_containers checks if a running crio instance
 # has no containers or pods
 function test_crio_wiped_containers() {
-	run crictl pods -v
-	[ "$status" -eq 0 ]
-	[ "$output" == "" ]
-
-	run crictl ps -v
-	[ "$status" -eq 0 ]
-	[ "$output" == "" ]
+	crictl pods -v
+	crictl ps -v
 }
 
 function test_crio_did_not_wipe_containers() {
-	run crictl pods -v
-	[ "$status" -eq 0 ]
-	[ "$output" != "" ]
+	crictl pods -v
```

Comment:
similar issue here -- there was a check that `crictl pods -v` actually printed something (i.e. output is non-empty), which this commit removes.

Reply:
fixed!